daemon/algod/api/algod.oas2.json

```diff
@@ -4111,6 +4118,18 @@
               "$ref": "#/definitions/SimulateTransactionGroupResult"
             }
           },
+          "unlimited-log": {
+            "description": "A boolean indicating whether this transaction is simulated with log usage unlimited",
+            "type": "boolean"
+          },
+          "max-log-calls": {
+            "description": "An integer indicating the limit of opcode log usage in simulating a transaction",
+            "type": "integer"
+          },
+          "max-log-size": {
+            "description": "An integer indicating the limit of byte number to log in simulating a transaction",
+            "type": "integer"
+          },
```

Comment:
I want to suggest an output format change based on two things: we're likely going to introduce many features which change eval, and I don't think it's useful to see the log limits unless you decide to enable unlimited logs.

So how about this:

```js
{
    "version": 1,
    "eval-changes": {
        "log-limits": { // this would be omitted if unlimited-logging is not enabled
            "max-log-calls": x,
            "max-log-size": y,
        },
        // future eval-changing features can live here
    },
    ...
}
```

I'm not sold on the `eval-changes` name, but I think it is valuable to collect all of the limit changes in one object.

Reply:
This design is even better. I was concerned of changing previous tests by spamming a bunch 1024 and 32 in expected results, and this design should improve a lot.

Reply:
I made some change in [eval-changes to capture changes in simulation](https://github.com/algorand/go-algorand/pull/5247/commits/7972596c025ef1216d8e53e3271584d492c46faa), though we haven't decided on if using `eval-changes` as its final name.

Reply:
hol up, I got it wrong, need to fix later

Reply:
Nice! I will resolve this